client/my-sites/site-settings/site-owner-transfer/use-check-site-transfer-eligibility.ts

```diff
@@ -19,19 +19,17 @@ export const useCheckSiteTransferEligibility = (
 	siteId: number | null,
 	options: UseMutationOptions< MutationResponse, MutationError, MutationVariables > = {}
 ) => {
-	const mutation = useMutation(
-		async ( { newSiteOwner }: MutationVariables ) =>
+	const mutation = useMutation( {
+		mutationFn: async ( { newSiteOwner }: MutationVariables ) =>
 			wp.req.post(
 				{
 					path: `/sites/${ siteId }/site-owner-transfer/eligibility`,
 					apiNamespace: 'wpcom/v2',
 				},
 				{ new_site_owner: newSiteOwner }
 			),
-		{
-			...options,
-		}
-	);
+		...options,
```

Comment:
Could it make sense to move this above `mutationFn` to prevent from accidentally overwriting it with `options`?